packages/ui/src/composables/useHover.ts

```diff
@@ -1,13 +1,17 @@
-import { ref, Ref, onMounted } from 'vue'
+import { ref, Ref, watch, onMounted } from 'vue'
 
 import { useHTMLElement, useEvent } from './'
 
-export function useHover (el?: Ref<HTMLElement | null | undefined>) {
+export function useHover (el?: Ref<HTMLElement | null | undefined>, disabled?: Ref<boolean>) {
   const isHovered = ref(false)
 
   const onMouseEnter = () => { isHovered.value = true }
   const onMouseLeave = () => { isHovered.value = false }
 
+  disabled && watch(disabled, (v) => {
```

Comment:
Is this necessary? Not seeing the case where this is needed? (why not keep it hovered?)

Reply:
Without it we will get this:

https://user-images.githubusercontent.com/55198465/186181812-3604e6c6-a74a-46ce-a260-0b524a6e7152.gif